article-list.html

```diff
@@ -37,49 +37,31 @@
                 <h1>All Articles</h1>
 
 				<!-- This is the category subheading that will change depending on which section of page user is on -->
-				<p class="subheading">Read articles about common skin issues</p>
+				<p class="subheading">Read articles about common skin questions</p>
 				<div class="image-first">
-					<a href="#"><img src="sample.png"></a>
+					<a href="fungal-acne.html"><img src="images/fungal_acne.jpg"></a>
 				</div>
 				<div class="title">
-					<a href="#">Article title will go here</a>
+					<a href="fungal-acne.html" class="first">What is Fungal Acne? (And how to treat it)</a>
+					<p>Learn what fungal acne is, how it's caused, and what you can do to treat it</p>
 				</div>
 
 				<div class="image">
-					<a href="#"><img src="sample.png"></a>
-					
+					<a href="clear-acne.html"><img src="images/acne.png"></a>
 				</div>
 				<div class="title">
-					<a href="#">Article title will go here</a>
-
+					<a href="clear-acne.html">The Best Products for Clearing Acne</a>
+					<p>Learn what some of the best products are out there for clearing acne</p>
 				</div>
 
-				<div class="image">

```

Comment:
In version 1, I originally had three empty boxes for skin issue articles as seen here. But since this would mean I would have to write many more articles, I decided to cut it down to 2 articles as seen in version 2.